core/src/actors/blocks_manager/handlers.rs

```diff
@@ -12,7 +12,11 @@ use witnet_util::error::WitnetError;
 
 use log::{debug, error};
 
-use super::messages::GetHighestBlockCheckpoint;
+use super::messages::{AddNewBlock, GetHighestBlockCheckpoint};
+use crate::actors::session::messages::AnnounceItems;
+use crate::actors::sessions_manager::messages::Broadcast;
+use crate::actors::sessions_manager::SessionsManager;
```

Comment:
perhaps these two `use` could be grouped into one? not sure what our code guidelines say about this